### PR TITLE
Update faker dependency to point to official community-maintained version

### PR DESCRIPTION
**Pull request by jasonbarry**
Addresses #1968

**Summary**
This PR replaces the compromised [faker](https://www.npmjs.com/package/faker) dependency with the official community-maintained [@faker-js/faker](https://www.npmjs.com/package/@faker-js/faker) dependency. The contents of the dependency (and version number) are the same. 

**Checklist**

- The basics
  - [ ] I tested these changes manually in my local or dev environment
- Tests
  - [x] Added or updated
  - [ ] N/A
- Event Tracking
  - [ ] I added event tracking and followed the event tracking guidelines
  - [x] N/A
- Error Reporting
  - [ ] I reported errors and followed the error reporting guidelines
  - [x] N/A

**Screenshots**
N/A

**Additional context**

### CODE REVIEW

File: packages/http/src/mocker/generator/__tests__/JSONSchema.spec.ts [hunk 1]
```diff
@@ -101,7 +101,7 @@ describe('JSONSchema generator', () => {
             meaning: {
               type: 'number',
               'x-faker': {
-                'random.number': {
+                'datatype.number': {
```
Comment (daniel-white) [inline packages/http/src/mocker/generator/__tests__/JSONSchema.spec.ts:104]: we'd like to ensure that the 5.x behavior is utilized - can this be reverted?

Comment (jasonbarry) [reply packages/http/src/mocker/generator/__tests__/JSONSchema.spec.ts]: @daniel-white done

**Comment by daniel-white** [comment]
one last thing: please pull from upstream - that'll fix links check and conflicting lockfile. thanks!

**Comment by jasonbarry** [comment]
@daniel-white just rebased, thanks!

**Comment by jasonbarry** [comment]
FYI @daniel-white I just pushed new commits to fix the failing tests.

**Review by chohmann — APPROVED** [review]

**Comment by chohmann** [comment]
This is included in prism 4.9.3! Thank you for contributing @jasonbarry!

**Pull request merged**